src/lib.rs

```diff
@@ -118,7 +118,33 @@ pub use defmt_macros::todo_ as unimplemented;
 /// [the manual]: https://defmt.ferrous-systems.com/macros.html
 pub use defmt_macros::panic_ as panic;
 
-/// todo
+/// This macro is roughly equivalent to `{Option,Result}::{expect,unwrap}` but invocation looks
```

Comment:
Maybe add a 1-line description at the top for nicer rustdoc output?

```suggestion
/// Unwraps an `Option` or `Result`, panicking if it is `None` or `Err`.
///
/// This macro is roughly equivalent to `{Option,Result}::{expect,unwrap}` but invocation looks
```